scripts/ci_moto_server.sh

```diff
@@ -2,4 +2,4 @@
 
 set -e
 pip install $(ls /moto/dist/moto*.gz)[server,all]
-moto_server -H 0.0.0.0 > /moto/server_output.log 2>&1
```

Comment:
this just made it easier to see the errors from the server side while running the docker container locally not in detached mode